src/statmodels.jl

```diff
@@ -259,10 +271,9 @@ In this formula, ``L`` is the likelihood of the model, ``L0`` that of the null m
 of the model (as returned by [`dof`](@ref)).
 """
```

Comment:
Also add parentheses after `log` in this docstring for consistency.

Reply:
Done!